src/services/price/index.ts

```diff
@@ -1,19 +1,98 @@
-import { PriceServiceError, UnknownPriceServiceError } from "@domain/price"
-import { getCurrentPrice } from "@services/realtime-price"
+import util from "util"
+import { credentials } from "@grpc/grpc-js"
+import {
+  PriceServiceError,
+  UnknownPriceServiceError,
+  PriceRange,
+  PriceNotAvailableError,
+} from "@domain/price"
+import { baseLogger } from "../logger"
+import { PriceHistory } from "./schema"
+import { PriceProtoDescriptor } from "./grpc"
+import { SATS_PER_BTC } from "@config/app"
+
+const priceUrl = process.env.PRICE_ADDRESS ?? "galoy-price"
+const pricePort = process.env.PRICE_PORT ?? "50051"
+const fullUrl = `${priceUrl}:${pricePort}`
+const priceClient = new PriceProtoDescriptor.PriceFeed(
+  fullUrl,
+  credentials.createInsecure(),
```

Comment:
I don't remember what `createInsecure()` in term of security but I guess something we might want to look into at some point